mailpoet/tests/integration/REST/Automation/Workflows/WorkflowsGetTest.php

```diff
@@ -4,11 +4,34 @@
 
 require_once __DIR__ . '/../AutomationTest.php';
 
+use MailPoet\Automation\Engine\Data\Step;
+use MailPoet\Automation\Engine\Data\Workflow;
+use MailPoet\Automation\Engine\Storage\WorkflowStorage;
 use MailPoet\REST\Automation\AutomationTest;
 
 class WorkflowsGetTest extends AutomationTest {
   private const ENDPOINT_PATH = '/mailpoet/v1/automation/workflows';
 
+  /** @var WorkflowStorage */
+  private $workflowStorage;
+
+  /** @var int[] */
+  private $userIds = [];
+
+  public function _before() {
+    parent::_before();
+    $this->workflowStorage = $this->diContainer->get(WorkflowStorage::class);
+    $userId = wp_insert_user([
+      'display_name' => 'test',
+      'roles' => ['editor'],
+      'email' => 'test@mailpoet.com',
+      'user_pass' => 'abc',
+      'user_login' => 'workflows-get-endpoint-test',
+    ]);
+    assert(is_numeric($userId));
```

Comment:
And in this case, we can also use `$this->assertIsNumeric(...)`.

Reply:
Addressed here: https://github.com/mailpoet/mailpoet/commit/c95dc0ffe7c959395012effca5c5885ad6f7ad2f#diff-b517c4a6320ddb55cbe3ff1aec93b5210bb4f3ed2845434e9d8eda69b105e6c6R31